src/email/index.js

```diff
@@ -58,13 +58,14 @@ const sendForgotPassword = async (userEmail, resetPasswordToken) => {
 const sendContactCheckInEmail = async (info) => {
   try {
     const entityLink = `${process.env.URL}/checkin/${info.entityId}?token=${info.token}`
-    const emailTitle = `${info.entityName} Check In`
+    const emailTitle = `${info.entityName} Healthcare Rollcall Check In`
```

Comment:
Should this be "HealthCare RollCall"?

Reply:
We're sticking with the traditional (nonCamel) capitalization